packages/headless/src/features/generated-answer/generated-answer-state.ts

```diff
@@ -1,12 +1,37 @@
 import {GeneratedAnswerCitation} from '../../api/generated-answer/generated-answer-event-payload';
 
 export interface GeneratedAnswerState {
+  /**
+   * Determines if the generated answer is loading.
+   */
   isLoading: boolean;
+  /**
+   * Determines if the generated answer is streaming.
+   */
   isStreaming: boolean;
+  /**
+   * The generated answer.
+   */
   answer?: string;
+  /**
+   * The citations used to generate the answer.
+   */
   citations: GeneratedAnswerCitation[];
+  /**
+   * Determines if the generated answer is liked, or upvoted by the end user.
```

Comment:
I'm not sure if "upvote", "downvote" is Reddit lingo or not haha. Maybe we can stick with "like" and "dislike" as that's what the property is called.

Reply:
nvm i see that's what we already have elsewhere. Maybe i even wrote it lol